custom-scorecard-tests/go.mod

```diff
@@ -0,0 +1,41 @@
+module main
```

Comment:
Please add a section to `.github/dependabot.yml` to update these deps. Should be something like:

```yaml
  - package-ecosystem: "gomod"
    directory: "/custom-scorecard-tests"
    schedule:
      interval: "weekly"
```

We should also put a check in the gh actions to ensure [go.mod and go.sum are in-sync](https://github.com/backube/volsync/blob/cac2570b518757f3b04076d49b7f4ce072cfa920/.github/workflows/operator.yml#L59-L68).